src/pages/workspaces/workspace/applications/AppLauncher.js

```diff
@@ -42,14 +42,7 @@ const ApplicationLauncher = _.flow(
     }),
     _.includes(runtimeStatus, usableStatuses) && cookieReady ?
       h(Fragment, [
-        application === 'RStudio' && h(PlaygroundHeader, [
-          'This feature is in early development. Your files are saved on your cloud environment but not to your workspace. We encourage you to frequently ',
-          h(Link, {
-            href: 'https://support.terra.bio/hc/en-us/articles/360037269472#h_822db925-41fa-4797-b0da-0839580a74da',
-            ...Utils.newTabLinkProps
-          }, ['back up your files manually']),
-          '.'
-        ]),
+        application === 'RStudio',
```

Comment:
This expression was used to control the visibility of the `PlaygroundHeader` via the `&&`, it doesn't make sense on its own. So if we're removing the content entirely, we should remove this expression as well.